src/profiler.cpp

```diff
@@ -1475,6 +1469,23 @@ void Profiler::printUsedMemory(Writer& out) {
     out << buf;
 }
 
+void Profiler::writeMetrics(Writer& out) {
+    out << "calltracestorage_bytes " << (u64) _call_trace_storage.usedMemory() << '\n';
```

Comment:
Maybe keep it in kb for readability?
Note that Linux typically reports memory stats in kB:
```
$ cat /proc/meminfo
MemTotal:       16204964 kB
MemFree:        15573588 kB
MemAvailable:   15567340 kB
Buffers:           13900 kB
Cached:           212264 kB
SwapCached:            0 kB
Active:            73628 kB
Inactive:         201364 kB
```
JVM does too:
```
$ jcmd `pgrep java` GC.heap_info
31427:
 garbage-first heap   total 256000K, used 2853K [0x0000000708a00000, 0x0000000800000000)
  region size 2048K, 1 young (2048K), 0 survivors (0K)
 Metaspace       used 83K, committed 320K, reserved 1114112K
  class space    used 5K, committed 128K, reserved 1048576K
```